src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html

```diff
@@ -67,6 +67,29 @@
     </ul>
 </app-page-options-section>
 
+<app-page-options-section sectionName="Find on Page">
+    <ul class="list-group">
+        <li class="list-group-item">                
+            <label class="small mx-1 fw-semibold">Comment:</label>
+            <div class="d-grid gap-2">
+                <div class="btn-group btn-group-sm" role="group" aria-label="Comment Navigation">
+                    <button type="button" class="btn btn-outline-secondary" (click)="commentThreadNavaigationEmitter.emit(CodeLineRowNavigationDirection.next)"><i class="bi bi-arrow-down"></i>Next</button>
+                    <button type="button" class="btn btn-outline-secondary" (click)="commentThreadNavaigationEmitter.emit(CodeLineRowNavigationDirection.prev)"><i class="bi bi-arrow-up"></i>Prev</button>
```

Comment:
can you add a tooltip so the complete message shows when hovering? i.e. `Previous`